semgrep-core/tests/rules/imported_metavariable_pattern.js

```diff
@@ -0,0 +1,4 @@
+import {bar as b} from 'foo';
+
+//ruleid: imported-metavariable-pattern 
+b();
```

Comment:
Do we have a test suite for autofixes? It would be nice to add an autofix test too for #4185.

Reply:
im getting the same autofix error even with this PR:
![image](https://user-images.githubusercontent.com/49291449/208778509-e911dee2-c19d-4a5e-a527-33a84de5e482.png)
im not sure this solves that problem